tests/test_quota.py

```diff
@@ -0,0 +1,53 @@
+from votekit.cvr_loaders import blt
+from votekit.election_types import STV, fractional_transfer
+from pathlib import Path
+import pytest
+
+BASE_DIR = Path(__file__).resolve().parent
+DATA_DIR = BASE_DIR / "data"
+
+
+def test_droop_default_parameter():
```

Comment:
the threshold tests should check that values computed are correct, not the election itself, here's some pseudo code to show what I mean 
`
def test_hare(): 
    election = STV(pp, fractional_transfer, seats=seats, quota="Droop")
    actual = 34.89
    assert threshold == election.threshold

`